<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LotWidget</class>
 <widget class="QWidget" name="LotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="mail-send">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableView" name="tableView">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="reloadButton">
       <property name="text">
        <string>Reload</string>
       </property>
       <property name="icon">
        <iconset theme="reload">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editButton">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset theme="tools-check-spelling">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>Quick search</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="placeCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>reloadButton</sender>
   <signal>clicked()</signal>
   <receiver>LotWidget</receiver>
   <slot>reloadList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>placeCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LotWidget</receiver>
   <slot>selectComm(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>LotWidget</receiver>
   <slot>editClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reloadList()</slot>
  <slot>editClicked()</slot>
  <slot>selectComm(int)</slot>
 </slots>
</ui>
